src/bench/bench_bitcoin.cpp

```diff
@@ -45,6 +49,14 @@ static std::vector<double> parseAsymptote(const std::string& str) {
     return numbers;
 }
 
+static uint8_t parsePriorityLevel(const std::string& str) {
```

Comment:
nit: I think we use `lowerCamelCase` only for interface method names (as per developer guidelines).
```suggestion
static uint8_t ParsePriorityLevel(const std::string& str) {
```